core/src/main/java/io/questdb/cairo/vm/api/MemoryCR.java

```diff
@@ -75,6 +75,11 @@ default int getInt(long offset) {
         return Unsafe.getUnsafe().getInt(addressOf(offset));
     }
 
+    default int getIPv4(long offset) {
```

Comment:
it would be good to check other memory implementation to have the same method

Reply:
Doesn't look like there are any other memory files that are appropriate to implement it in